sdk/js/src/relayer/__tests__/wormhole_relayer.ts

```diff
@@ -0,0 +1,690 @@
+  "eb94ff04accbfc8195d44b45e7c7da4c6993b2fbbfc4ef166a7675a905df9891",
+  "053a6527124b309d914a47f5257a995e9b0ad17f14659f90ed42af5e6e262b6a",
+  "3fbf1e46f6da69e62aed5670f279e818889aa7d8f1beb7fd730770fd4f8ea3d7",
+  "53b05697596ba04067e40be8100c9194cbae59c90e7870997de57337497172e9",
+  "4e95cb2ff3f7d5e963631ad85c28b1b79cb370f21c67cbdd4c2ffb0bf664aa06",
+  "01b8c448ce2c1d43cfc5938d3a57086f88e3dc43bb8b08028ecb7a7924f4676f",
+  "1db31a6ba3bcd54d2e8a64f8a2415064265d291593450c6eb7e9a6a986bd9400",
+  "70d8f1c9534a0ab61a020366b831a494057a289441c07be67e4288c44bc6cd5d",
+];
+const GUARDIAN_SET_INDEX = 0;
+
+
+// for signing wormhole messages
+const guardians = new MockGuardians(GUARDIAN_SET_INDEX, GUARDIAN_KEYS);
+
+const GOVERNANCE_EMITTER_ADDRESS = new PublicKey(
+  "11111111111111111111111111111115"
+);
+
+
+// for generating governance wormhole messages
+const governance = new GovernanceEmitter(
+GOVERNANCE_EMITTER_ADDRESS.toBuffer().toString("hex")
+);
+
+const guardianIndices = env=='ci'?[0,1]:[0];
+
+const getStatus = async (txHash: string): Promise<string> => {
+    console.log(env);
+    console.log(sourceChainId);
+    console.log(txHash);
+  const info = (await relayer.getWormholeRelayerInfo(
+      sourceChainId,
+      txHash,
+      { environment: network, ...infoRequestOptionalParams }
+    )) as relayer.DeliveryInfo;
+  return  info.targetChainStatus.events[0].status;
+}
+
+const testSend = async (payload: string, sendToSourceChain?: boolean, notEnoughValue?: boolean): Promise<string> => {
+  const value = await sourceCoreRelayer.quoteGas(
+      sendToSourceChain ? sourceChainId : targetChainId,
+      notEnoughValue ? 10000 : 500000,
+      await sourceCoreRelayer.getDefaultRelayProvider()
+    );
+    console.log(`Quoted gas delivery fee: ${value}`);
+    const tx = await sourceMockIntegration.sendMessage(
+      payload,
+      sendToSourceChain ? sourceChainId : targetChainId,
+     sendToSourceChain ? sourceMockIntegrationAddress : targetMockIntegrationAddress,
+      { value, gasLimit: 500000 }
+    );
+    console.log("Sent delivery request!");
+    const rx = await tx.wait();
+    console.log("Message confirmed!");
+
+    return tx.hash;
+}
+
+const testForward = async (payload1: string, payload2: string, notEnoughExtraForwardingValue?: boolean): Promise<string> => {
+  const value = await sourceCoreRelayer.quoteGas(
+      targetChainId,
+      500000,
+      await sourceCoreRelayer.getDefaultRelayProvider()
+    );
+    const extraForwardingValue = await targetCoreRelayer.quoteGas(
+      sourceChainId,
+      notEnoughExtraForwardingValue ? 10000 : 800000,
+      await targetCoreRelayer.getDefaultRelayProvider()
+    );
+    console.log(`Quoted gas delivery fee: ${value.add(extraForwardingValue)}`);
+
+    const furtherInstructions: ethers_contracts.MockRelayerIntegration.FurtherInstructionsStruct = {
+      keepSending: true,
+      newMessages: [payload2, "0x00"],
+      chains: [sourceChainId],
+      gasLimits: [500000],
```

Comment:
I keep seeing 10000 and 500000 pop up in the code - looks like that should be a variable with a name that says what the value is

Reply:
done